src/ApplicationIO.cpp

```diff
@@ -54,16 +54,14 @@ namespace geopm
 {
     constexpr size_t ApplicationIOImp::M_SHMEM_REGION_SIZE;
 
-    ApplicationIOImp::ApplicationIOImp(const std::string &shm_key)
-        : ApplicationIOImp(shm_key,
```

Comment:
Seems like the shm_key handling here was never used previously?  I'm trying to recall why we plumbed this all in.

Reply:
Maybe we had intended to pass it through and forgot to fix this?  The other objects that use shared memory independently look up the environment variable for the shm_key so it should actually be the same in practice.